src/test/java/com/cloudogu/smeagol/wiki/infrastructure/TimeBasedPullChangesStrategyTest.java

```diff
@@ -0,0 +1,20 @@
+package com.cloudogu.smeagol.wiki.infrastructure;
+
+import com.cloudogu.smeagol.wiki.domain.WikiId;
+import org.junit.Test;
+
+import static com.cloudogu.smeagol.wiki.DomainTestData.WIKI_ID_42;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TimeBasedPullChangesStrategyTest {
+
```

Comment:
Maybe we should add a test: shouldPullNot ?

Reply:
The test for should pull contains both shouldPull returning true and false.